### DISCUSSION TITLE
Belgium marketplace support for GET_SALES_AND_TRAFFIC_REPORT

### DISCUSSION THREAD

**Discussion by tho-masn** [amzn/selling-partner-api-models#3492] [General]
Hi there,

Seems like some of the SP-API reports are not implemented for the Belgium marketplace (AMEN7PMS3EDWL).

Currently I'm getting the following response, when I'm trying to request the GET_SALES_AND_TRAFFIC_REPORT report for the Belgium marketplace: `Error: Report rejected for this client. Some of the marketplaces are not allowed for this Report `

Is there any ETA when this will arrive? I can find any information on what is supported for the Belgium marketplace. Is there any official documentation I've missed?

Thanks!

**Comment by discover59** [comment]
I have same issue. I tried it after reauthorization and it didn't work.

**Comment by chrizi42** [comment]
A1MQXOICRS2Z7M and AHRY1CZE9ZY4H does not work either. The documentation for this report type says nothing about a limitation for certain marketplaces: https://developer-docs.amazon.com/sp-api/docs/report-type-values#seller-retail-analytics-reports

**Comment by nassuf** [comment]
Hey
I also have the same issue with Belgium and GET_SALES_AND_TRAFFIC_REPORT report.
We tried to re-authorise etc, nothing works.

**Comment by moltar** [comment]
Same here. Does not work for BE.

@amzn could you please document this report better?

Thanks.

**Reply by jamesp26** [reply]
Would be nice for this to get resolved.

**Comment by ritikacd** [comment]
A known issue for customers in the Belgium marketplace requesting the [GET_SALES_AND_TRAFFIC_REPORT](https://github.com/amzn/selling-partner-api-models/blob/main/schemas/reports/sellerSalesAndTrafficReport.json) has been fixed. Integrators can now successfully request the Sales and Traffic report in BE.  

Let us know if you still have this issue. 

Thanks,
Ritika Chaudhary
Selling Partner API Team

**Comment by nguyentoanit** [comment]
Anyone gets an `InvalidInput` error after this fix: 

> A known issue for customers in the Belgium marketplace requesting the [GET_SALES_AND_TRAFFIC_REPORT](https://github.com/amzn/selling-partner-api-models/blob/main/schemas/reports/sellerSalesAndTrafficReport.json) has been fixed. Integrators can now successfully request the Sales and Traffic report in BE.
> 
> Let us know if you still have this issue.
> 
> Thanks, Ritika Chaudhary Selling Partner API Team


```json
{
    "errors": [
        {
            "code": "InvalidInput",
            "message": "Marketplace AMEN7PMS3EDWL is invalid",
            "details": ""
        }
    ]
}
```
I got an error message when requesting to Belgium marketplace. 🤔
